Code/Framework/AzCore/Platform/Windows/AzCore/IO/Streamer/StreamerConfiguration_Windows.cpp

```diff
@@ -318,7 +318,7 @@ namespace AZ::IO
                     deviceName += driveIt;
                     deviceName.erase(deviceName.length() - 1); // Erase the slash.
 
-                    HANDLE deviceHandle = ::CreateFile(
+                    HANDLE deviceHandle = ::CreateFileA(
```

Comment:
Shouldn't this be the W version?

Reply:
yeah, I didnt update all cases, we will have to do another pass to continue replacement... there are still a handful
I will check if I can tackle those, but need to draw the line here since the UNICODE replacement ended up being another monster and I am interleaving it with the CryString removal